XamDialogs

newky2k opened the pull request:
Migrated to Nuget and CI

Commit:
Added code from old repo

Added code from old repo

Commit:
Added build.cake and renamed src folder

Added build.cake and renamed src folder

Commit:
Added external dependency info and  nuget section to build.cake

Added external dependency info and  nuget section to build.cake

Commit:
updated urls to use FWLink and copyrights

updated urls to use FWLink and copyrights

bholmes commented on iOS/XamDialogs/component/component.yaml:
```diff
@@ -0,0 +1,30 @@
+  - screenshots/Screen1.png
+getting-started: GettingStarted.md
+version: 1.1
+samples: 
```
Update sample to use Nuget with removeProject and instalNuget.  local-nuget-repo is needed too.

Commit:
Fixes for PR Review

Fixes for PR Review

bholmes committed:
[XamDialogs] Adding to manifest.json

bholmes committed:
[XamDialogs] Fixes to PR merge

 - Change license to MIT
 - Rename Nuget to Xamarin.XamDialogs
 -  Add THIRD-PARTY-NOTICES.txt to NuGet and Component xam file

bholmes committed:
Merge branch 'master' into dh_XamDialogs
